coriolis/osmorphing/osmount/base.py

```diff
@@ -389,17 +389,22 @@ def _find_and_mount_root(self, devices):
         os_root_device = self._find_dev_with_contents(
             devices, all_files=files)
 
+        user_tips = (
+            "Please check the source VM's contents of '/etc/fstab' or try "
+            "using a more recent version of the worker's OS")
         if os_root_device is None:
             raise exception.OperatingSystemNotFound(
-                "root partition not found")
+                "Worker could not identify the migrated VM's root partition. "
+                "%s" % user_tips)
 
         try:
             tmp_dir = self._exec_cmd('mktemp -d').decode().splitlines()[0]
             self._exec_cmd('sudo mount %s %s' % (os_root_device, tmp_dir))
             os_root_dir = tmp_dir
         except Exception:
             self._event_manager.progress_update(
-                "Failed to mount root device '%s'" % os_root_device)
+                "Failed to mount root device '%s'. %s" % (
```

Comment:
"Exception occurred while Coriolis was attempting to mount the root device (%s) of the OS being migrated for OSMorphing. Please ensure that the source VM's root partition(s)are using a filesystem type and version which is supported by the OS images used for the OSMorphing minion machine. Also ensure that the source VM's mountpoint declarations in '/etc/fstab' are all correct, and are declared using '/dev/disk/by-uuid/' or 'UUID=' notation. If all else fails, please retry while using an OSMorphing minion machine image which is the same OS release as the VM being migrated. Error was %s" % (os_root_device, str(ex))